.github/workflows/deploy-database.yml

```diff
@@ -24,3 +25,20 @@ jobs:
         run: npx supabase db push --dry-run
       - name: Apply the migration
         run: npx supabase db push
+
+  reset-dev-database-data:
+    name: Reset data in dev database
+    if: github.event_name == 'workflow_dispatch'
```

Comment:
This makes sure that we can only run this as a manual step on the default branch (main)